api/v1alpha1/storageconsumer_types.go

```diff
@@ -58,8 +61,6 @@ type StorageConsumerStatus struct {
 	GrantedCapacity resource.Quantity `json:"grantedCapacity,omitempty"`
 	// CephObjects provide details of ceph created objects required for external storage
 	CephObjects CephObjectsSpec `json:"cephObjects,omitempty"`
-	// ConnectionDetails holds the reference to secret containing external connection details
-	ConnectionDetails *v1.SecretKeySelector `json:"connectionDetails,omitempty"`
```

Comment:
This should probably be a separate PR or commit? Doesn't look related to adding new status fields.